src/components/Modal/Modal.tsx

```diff
@@ -30,6 +30,7 @@ interface Modal<P> extends React.FunctionComponent<P> {
 
 const { block, elem } = bem('Modal', styles);
 
+// eslint-disable-next-line @typescript-eslint/no-redeclare
```

Comment:
This is a temporary workaround: would get a linting error without.

Reply:
Please rename the Modal interface above to something else. That should do the trick.